.eslintignore

```diff
@@ -0,0 +1,7 @@
+node_modules
+
+## babel, jest, prettier
+*.config.js
+
+## test files
+*.test.js
```

Comment:
I actually find adding eslint to tests helpful. There are small things during development it catches like whether or not a variable is defined etc.

Reply:
Yeah let's add it back